source/code/source/playfab/PlayFabIOSHttpPlugin.mm

```diff
@@ -268,7 +268,14 @@ void Cancel()
                                                                        NSString* body = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
 
                                                                        CallRequestContainer& requestContainer = request->RequestContainer();
-                                                                       requestContainer.responseString = [body UTF8String];
+                                                                       if(body != nil)
+                                                                       {
+                                                                           requestContainer.responseString = [body UTF8String];
+                                                                       }
+                                                                       else
+                                                                       {
+                                                                           requestContainer.responseString = "PlayFabIOSHttpPlugin failed to parse body as it is nil";
```

Comment:
We should not be setting a response string to a non empty or nil value. Do we know why it is crashing, is the length in addition to it being nil?

Reply:
To be more specific, adding in our hint to the response is not generally acceptable.